src/app/my-notifications/components/my-notifications-show/my-notifications-show.component.html

```diff
@@ -0,0 +1,79 @@
+<div
+  class="notifications-container"
+  *erzLet="{
+    notifications: notifications$ | async,
+    numNotifications: (notifications$ | async)?.length || 0,
```

Comment:
Naming: Vielleicht "notificationCount"? Wir verwenden diesen Term in anderen Modulen.